### PR TITLE
[Misc] Update version

### PR DESCRIPTION
**Pull request by Accelerator1996**

### CODE REVIEW

**Comment by Accelerator1996** [comment]
test

File: version.txt [hunk 1]
```diff
@@ -1 +1 @@
-21.0.4.0.4+7-GA
\ No newline at end of file
+21.0.5.0.5+9-GA
```
Comment (Accelerator1996) [inline version.txt:1]: test

Comment (Accelerator1996) [inline version.txt:1]: test

Comment (Accelerator1996) [inline version.txt:1]: test

Comment (Accelerator1996) [inline version.txt:1]: diff --git a/version.txt b/version.txt
index dec00a8e732..78a582d5ce8 100644
--- a/version.txt
+++ b/version.txt
@@ -1 +1 @@
-21.0.5.0.5+9-GA
\ No newline at end of file
+21.0.4.0.4+7-GA
\ No newline at end of file

Comment (Accelerator1996) [inline version.txt:1]:
```
diff --git a/version.txt b/version.txt
index dec00a8e732..78a582d5ce8 100644
--- a/version.txt
+++ b/version.txt
@@ -1 +1 @@
-21.0.5.0.5+9-GA
\ No newline at end of file
+21.0.4.0.4+7-GA
\ No newline at end of file
```

**Review by kuaiwei — APPROVED** [review]
LGTM

**Review by sendaoYan — APPROVED** [review]
/LGTM

**Pull request merged**